OsmAnd/src/net/osmand/plus/poi/PoiFiltersHelper.java

```diff
@@ -121,23 +116,18 @@ public PoiUIFilter getCustomPOIFilter() {
 
 	public void prepareTopWikiFilter(@NonNull PoiUIFilter wiki) {
```

Comment:
We need to do it via OsmAndPlugin.prepareExtraTopPoiFilters and there specify all what we need for this feature.